### DISCUSSION TITLE
Navigating in RStudio preview opens file

### DISCUSSION THREAD

**Discussion by JosiahParry** [quarto-dev/quarto-cli#3312] [Q&A]
I'm not sure how to repro this without providing the entirey of my site. 

When navigating a website preview in the viewer pane in RStudio, clicking on a link or post will open the source file. This, to me, is unexpected behavior. Is this a _feature_ of quarto? If so, how can I deactivate it?

https://user-images.githubusercontent.com/13534508/201667887-a2e16be5-d9b2-46a1-b811-dcb3de35b4fa.mov

**Comment by JosiahParry** [comment]
Note that this does not happen when using the browser preview.

**Comment by JosiahParry** [comment]
This is embarassing. It's the button `Sync Editor` that i must've clicked without realizing :) 

I don't know how to close a discussion post :P

**Comment by mcanouil** [comment]
I believe this would probably be better on https://github.com/rstudio/rstudio since it is very likely to be an issue with the IDE rather than Quarto.
